app/services/transaction_search/searcher.rb

```diff
@@ -10,6 +10,7 @@ class Searcher
     cattr_accessor(:default_searchers) do
       [
         TransactionSearch::AccountSearcher,
+        TransactionSearch::AccountTypeSearcher,
```

Comment:
Just a gut check, is it ok for all searches to immediately include this new `AccountTypeSearcher`?